ortho_seq_code/tests/test_orthoseqs.py

```diff
@@ -122,8 +122,11 @@ def test_nucleotide_first_order(
         basename = os.path.basename(nucleotide_params_first_order.seqs_filename)
         basename_pheno = os.path.basename(nucleotide_params_first_order.pheno_filename)
         expected_path = os.path.join(nucleotide_first_order_data_dir, basename + ".npz")
-        obtained_path = os.path.join(location, basename + ".npz")
-        print("Obtained Path:")
+        obtained_path = (
+            os.path.join(location, basename + ".npz")
+            .replace("\n", "")
```

Comment:
Not sure what these replaces are exactly doing. Do you have a next line in the file when it's saved? That does not seem feasible to me

Reply:
I'm trying to modify the obtained path such that pytest likes it, but so far nothing has changed.